install/kubernetes/cilium/templates/cilium-operator/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if and .Values.operator.enabled .Values.operator.podDisruptionBudget.enabled }}
+apiVersion: policy/v1beta1
```

Comment:
while policy/v1beta1 is available till kube 1.24, I think it's better to determine PDB api version based on kube version, something similar to the below

https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/templates/_helpers.tpl#L80-L88

Reply:
have fixed.